### PR TITLE
restructure realtime server for clarity

### PR DESCRIPTION
**Pull request by Connoropolous**

### CODE REVIEW

File: frontend/src/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // create global references
 import _ from 'underscore'
+import io from 'socket.io-client'
 import Metamaps from './Metamaps'
 
 window._ = _
+window.io = io
```
Comment (devvmh) [inline frontend/src/index.js]: why on window?

File: realtime/global.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  // server receivable, client sendable
+  REQUEST_LIVE_MAPS,
+  JOIN_MAP,
+  LEAVE_MAP,
+  UPDATE_TOPIC,
+  DELETE_TOPIC,
+  UPDATE_SYNAPSE,
+  DELETE_SYNAPSE,
+  UPDATE_MAP
+} from '../frontend/src/Metamaps/Realtime/events'
```
Comment (devvmh) [inline realtime/global.js]: oh man, code sharing :D

**Comment by Connoropolous** [comment]
I've been running this with 
`DEBUG=socket.io* babel-node realtime/realtime-server.js` locally for testing

the reason it needs babel-node is because of the imports. no need to run it with logging on the server

File: frontend/src/index.js (hunk 1 above)
Comment (Connoropolous) [inline frontend/src/index.js]: actually, we don't need it, now that socketioconnection is being imported

Comment (Connoropolous) [inline frontend/src/index.js]: will remove

**Comment by Connoropolous** [comment]
#570 and #712

**Pull request merged**